CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.10.6
+
+<!-- generated comparing v5.10.5..master -->
+
+_Sep 19, 2022_
+
+A big thanks to the 11 contributors who made this release possible.
+This release was mostly about 🐛 bug fixes and 📚 documentation improvements.
+
+### `@mui/material@5.10.6`
+
+- [TextField] Fix conflict with `Bootstrap` even when label is not defined (#34343) @ZeeshanTamboli
+
+### `@mui/joy@5.0.0-alpha.46`
+
+- [Joy][button] Replace `start/endIcon` prop with `start/endDecorator` (#34288) @hbjORbj
```

Comment:
This is a breaking change. Do we need to add `BREAKING CHANGE` for alpha packages?

Reply:
Yes, we should add a BREAKING CHANGE section, to allow deverlopers to adopt the breaking change easily.

Also one note on the PR description, we decided to include the component name first, and then the package, so this should be:

```suggestion
- [button][Joy] Replace `start/endIcon` prop with `start/endDecorator` (#34288) @hbjORbj
```